src/cmake/thirdparty/SetupAxomThirdParty.cmake

```diff
@@ -107,10 +107,11 @@ endif()
 #------------------------------------------------------------------------------
 if (MFEM_DIR)
     include(cmake/thirdparty/FindMFEM.cmake)
-    blt_register_library( NAME      mfem
-                          INCLUDES  ${MFEM_INCLUDE_DIRS}
-                          LIBRARIES ${MFEM_LIBRARIES}
-                          TREAT_INCLUDES_AS_SYSTEM ON)
+    blt_import_library( NAME       mfem
+                        INCLUDES   ${MFEM_INCLUDE_DIRS}
+                        LIBRARIES  ${MFEM_LIBRARIES}
+                        TREAT_INCLUDES_AS_SYSTEM ON
+                        EXPORTABLE ON)
```

Comment:
Can you please discuss `EXPORTABLE` here, vs. `BLT_EXPORT_THIRDPARTY` in the options above?
I suppose we always want to export MFEM (when `MFEM_DIR`) but only sometimes want to export BLT's thirdparties ?